fairlearn/reductions/_moments/utility_parity.py

```diff
@@ -272,8 +302,25 @@ class DemographicParity(UtilityParity):
 
     .. math::
       P[h(X) = 1 | A = a, C = c] = P[h(X) = 1 | C = c] \; \forall a, c
-    """
+<<<<<<< Updated upstream
+        
+     References
+     ----------
+     .. [1] A. Agarwal, A. Beygelzimer, M. Dudík, J. Langford,
+        and H. Wallach, “A Reductions Approach to Fair Classification,” arXiv.org, 16-Jul-2018.
+        [Online]. Available: https://arxiv.org/abs/1803.02453. [Accessed: 05-Apr-2021].
+           
+=======
+    
+    References
+    ----------
+    .. [1] A. Agarwal, A. Beygelzimer, M. Dudík, J. Langford,
+       and H. Wallach, “A Reductions Approach to Fair Classification,” arXiv.org, 16-Jul-2018.
+       [Online]. Available: https://arxiv.org/abs/1803.02453. [Accessed: 05-Apr-2021].
```

Comment:
Regarding the format: do we need the "Accessed: <date>" part? I'm personally not a huge fan but I realize this is very subjective. I know that this is very much a good practice with articles on the web since they could disappear or get changed anytime. On arXiv I'm not too worried since they track versions. Wdyt? @fairlearn/fairlearn-maintainers

Reply:
I agree @romanlutz! I can see why it would be good practice in a static document (e.g., backing up a claim in a paper), but given our goal (provide reference materials to learn more) and medium (dynamic web page) it doesn't really matter until what date it was still available - once it is broken it's useless anyways.